cubequery/api_server.py

```diff
@@ -160,16 +185,17 @@ def normalise_task_info(info):
     result = []
 

```

Comment:
need to add a check for info here as you have done above.

Reply:
Weirdly you get different response objects from JobTastic when asking for a single task and asking for all the tasks. 
You always get an empty list back in the case of no jobs when asking for lots of them.
You get none back when asking for a single task.

The internal structure is also slightly different meaning I need these two separate function, which is annoying.